app/src/main/java/be/ugent/zeus/hydra/adapters/SectionPagerAdapter.java

```diff
@@ -24,6 +25,8 @@ public Fragment getItem(int position) {
                 return new HomeFragment();
             case 1:
                 return new MinervaFragment();
+            case 4:
+                return new ComingSoonFragment();
```

Comment:
Maybe it would be better to set the ComingSoonFragment as default case.
So that we only switch to the shipping fragments :)